policy/controller.go

```diff
@@ -15,9 +15,9 @@ import (
 	"golang.org/x/exp/constraints"
 )
 
-// policy.SetVariations needs to print the variations in a
+// policy.SetOutputs needs to print the variations in a
 // well-defined order for TestOutput
-func sortedKeys[K constraints.Ordered, V any](m map[K]V) []K {
+func sortedVariations[K constraints.Ordered, V any](m map[K]V) []K {
```

Comment:
This is a generic function that can operate on any ordered type. Renaming it is not required.